test/common/services/security/user.ts

```diff
@@ -62,6 +62,6 @@ export class User {
         `Expected status code of 204, received ${status} ${statusText}: ${util.inspect(data)}`
       );
     }
-    this.log.debug(`deleted user ${username}`);
+    // this.log.debug(`deleted user ${username}`);
```

Comment:
should it be re-enabled or completely removed?

Reply:
re-enabled it